intelligence/views.py

```
@@ -40,6 +41,8 @@ class AISummaryViewSet(viewsets.ModelViewSet):
     """ViewSet for AI summaries with GapGPT/OpenAI integration"""
     queryset = AISummary.objects.all().select_related('patient', 'content_type')
     permission_classes = [IsAuthenticated]
+    throttle_classes = [ScopedRateThrottle]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)